src/v/cluster/metrics_reporter.cc

```diff
@@ -416,6 +417,8 @@ void rjson_serialize(
         rjson_serialize(w, d);
     }
     w.EndArray();
+    w.Key("storage_space_alert");
```

Comment:
Should probably give @0x5d a heads up about this.  I don't know if there are any plans to use these metrics for alerting -- the prometheus output is the important one.

Reply:
I was going to ask about the consumption side.. Thank you.

Reply:
This currently wouldn't need further changes in the service. I.e. it would get stored along with the rest of the JSON and wouldn't break anything. If we'd like to build alerts, however, then yes.